core/src/main/resources/reference.conf

```diff
@@ -252,9 +252,18 @@ akka.persistence.cassandra {
     # Max time to buffer events for before writing.
     # Larger values will increase cassandra write efficiency but increase the delay before
     # seeing events in EventsByTag queries.
-    # Setting this to 0 means that tag writes will be written immediately but will still be asynchronous
-    # with respect to the PersistentActor's persist call.
-    flush-interval = 250ms
+    # Setting this to 0 means that tag writes will be written immediately. Batching will still happen
```

Comment:
Just came across this PR when debugging an issue: https://discuss.lightbend.com/t/akka-persistence-cassandra-unrecoverable-failures-when-tag-writes-buffer-gets-too-big/7643

We're using version 1.0.1 of this, would upgrading to 1.0.4 help resolve the issue we're seeing?

Reply:
Replied on the thread